.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
 commands:
   install_macos_dependencies:
     steps:
-      - run: brew install protobuf cmake ccache libtool boost-python libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
+      - run: brew install protobuf cmake ccache coreutils libtool boost-python libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
```

Comment:
This change isn't needed now that the checksum is done in pure cmake

Reply:
I left it in since it was listed as a build requirement on osx in readme: https://github.com/valhalla/valhalla#building-from-source

But I can remove it, fix pushed.